bin/run_tests.sh

```diff
@@ -117,10 +117,12 @@ else
                     test/models/branding_group_test.rb
                     test/models/synthetic_branding_group_populator_test.rb
                     test/models/admin_configuration_test.rb
+                    test/integration/synthetic_study_populator_test.rb
```

Comment:
I don't think this line belongs in this PR

Reply:
This was already on `development` when I created this branch to begin working.  I was going to remove it, but given that this particular commit does not touch any production code (it is completed confined to the `test` environment, and is only invoked in CI via Travis) it seemed simpler to leave it in rather than remove it.

Reply:
7539771